### PR TITLE
[docs](function) support aggregrate function regr_sxx regr_sxy regr_syy

### PR DESCRIPTION
**Pull request by wyxxxcat**
https://github.com/apache/doris/pull/39187

### CODE REVIEW

File: i18n/zh-CN/docusaurus-plugin-content-docs/version-3.0/sql-manual/sql-functions/aggregate-functions/regr_sxy.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+under the License.
+-->
+
+## REGR_SXY
+### Description
+#### Syntax
+
+` double regr_sxy(y, x)`
+
+计算两个变量 'X' 和 'Y' 的偏差乘积和，即 'X' 和 'Y' 相对于各自平均值的偏差的乘积之和
```
Comment (zclllyybb) [inline i18n/zh-CN/docusaurus-plugin-content-docs/version-3.0/sql-manual/sql-functions/aggregate-functions/regr_sxy.md:33]: use ``, not ''

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/sql-manual/sql-functions/aggregate-functions/regr_sxx.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+## REGR_SXX
+### Description
+#### Syntax
+
+` double regr_sxx(y, x)`
```
Comment (HappenLee) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/sql-manual/sql-functions/aggregate-functions/regr_sxx.md:31]: need desc the `y, x` type, `doulbe` or other type?

**Review by KassieZ — COMMENTED** [review]
1. title:"REGR_SXX" and ## REGR_SXX is duplicate, please delete ## REGR_SXX and also upgrade all level-3 title to level-2
2. Please capitalize Example, Keywords
3. Does v2.1 / v2.0 also support the function? if yes, please also update it

**Comment by wyxxxcat** [comment]
> 1. title:"REGR_SXX" and ## REGR_SXX is duplicate, please delete ## REGR_SXX and also upgrade all level-3 title to level-2
> 2. Please capitalize Example, Keywords
> 3. Does v2.1 / v2.0 also support the function? if yes, please also update it

Done,the function only support master and v3.0

**Pull request merged**